packages/serialization/src/serializers/errors/InvalidUrlString.ts

```diff
@@ -0,0 +1,8 @@
+
+export default class InvalidUrlString extends Error
+{
+    constructor(urlString: string)
+    {
+        super(`Invalid url string: '${urlString}'`);
```

Comment:
```suggestion
        super(`The url string '${urlString}' is invalid`);
```
Same format as the InvalidClass error